# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1429704323 18000
#      Wed Apr 22 07:05:23 2015 -0500
# Node ID 5f0cb9bb6a50ba1e75300e08643d64d0a6d8b34d
# Parent  c36f2b0836130baecf5902b79794f23007493997
use png instead of jpg as default format for copying plots to clipboard

* Figure.cc (Figure::editCopy): Initialize format to "png".

diff --git a/libgui/graphics/Figure.cc b/libgui/graphics/Figure.cc
--- a/libgui/graphics/Figure.cc
+++ b/libgui/graphics/Figure.cc
@@ -846,17 +846,17 @@ void
 Figure::fileCloseFigure (void)
 {
   qWidget<QMainWindow> ()->close ();
 }
 
 void
 Figure::editCopy (bool /* choose_format */)
 {
-  QString format = "jpg";
+  QString format = "png";
 
 #if 0
 
   // FIXME: allow choice of image formats.
 
   if (choose_format)
     {
       QFileInfo finfo (file);
